dask/dataframe/io/tests/test_sql.py

```diff
@@ -176,7 +176,7 @@ def test_npartitions(db):
         "test",
         db,
         columns=list(df.columns),
-        bytes_per_chunk=2 ** 30,
+        bytes_per_chunk="2 GiB",
```

Comment:
Could you test both of these cases, to ensure that we don't regress on accepting integers?

Reply:
I flopped around some of the current uses of this keyword in the test suite to strings while leaving some others as integers.  I think that we're covered in both cases, even if there isn't an explicit test.  I can make an explicit test though if preferred.

Reply:
As long as we have some coverage for passing integers I think we're good.